packages/apollo-server/src/express.ts

```diff
@@ -4,6 +4,7 @@ import { json } from 'body-parser';
 import { createServer, Server as HttpServer } from 'http';
 import { graphqlExpress } from 'apollo-server-express';
 import graphiql from 'graphql-playground-middleware-express';
```

Comment:
How about renaming this `import` to `gui` instead of `graphiql`, since it's no longer `graphiql`?